app/src/main/res/layout/activity_reservation.xml

```diff
@@ -0,0 +1,164 @@
+            android:layout_height="wrap_content">
+            <TextView
+                android:id="@+id/text_view_check"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/main_activity_text_view_margin_top"
+                android:text="@string/text_view_reservation_check_in"
+                android:textColor="@color/main_activity_white_text"
+                android:textSize="@dimen/main_activity_text_size"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent" />
+
+            <Button
+                android:id="@+id/button_check_in"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/activity_text_view_margin"
+                android:text="@string/text_button_check_in"
+                app:layout_constraintTop_toBottomOf="@id/text_view_check"
+                app:layout_constraintBottom_toTopOf="@id/text_view_check_in_from"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toLeftOf="@id/text_view_check_out"/>
+
+            <Button
+                android:id="@+id/button_check_out"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/activity_text_view_margin"
+                android:text="@string/text_button_check_out"
+                app:layout_constraintTop_toBottomOf="@id/text_view_check"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintLeft_toRightOf="@id/button_check_in"/>
+
+            <TextView
+                android:id="@+id/text_view_check_in"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/main_activity_text_view_margin_top"
+                app:layout_constraintBottom_toBottomOf="@id/text_view_check_in_from"
+                app:layout_constraintLeft_toRightOf="@id/text_view_check_in_from"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/button_check_out" />
+
+            <TextView
+                android:id="@+id/text_view_check_out"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/main_activity_text_view_margin_top"
+                android:layout_marginBottom="@dimen/activity_text_view_margin"
+                app:layout_constraintBottom_toBottomOf="@id/text_view_check_out_to"
+                app:layout_constraintLeft_toRightOf="@id/text_view_check_out_to"
```

Comment:
Please use start and end instead of left and right

Reply:
Done